modules/logging-forwarding-azure.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// logging/configuring-log-forwarding.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="<logging-forwarding-azure>_{context}"]
+= Forwarding to Azure Monitor Logs
+With {logging} 5.9 and higher, you can forward logs to link:https://learn.microsoft.com/en-us/azure/azure-monitor/logs/data-platform-logs[Azure Monitor Logs] in addition to, or instead of, the default log store. This functionality is provided by the link:https://vector.dev/docs/reference/configuration/sinks/azure_monitor_logs/[Vector Azure Monitor Logs sink].
+
+.Prerequisites
+
+* You are familiar with how to administer and create a `ClusterLogging` custom resource (CR) instance.
+* You are familiar with how to administer and create a `ClusterLogForwarder` CR instance.
+* You understand the `ClusterLogForwarder` CR specifications.
+* You have basic familiarity with Azure services.
+* You have an Azure account configured for Azure Portal or Azure CLI access.
+* You have obtained your Azure Monitor Logs primary or the secondary security key.
+* You have determined which log types to forward.
+
+To enable log forwarding to Azure Monitor Logs via the HTTP Data Collector API:
+
+. Create a secret with your shared key:
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: my-secret
+  namespace: openshift-logging
+type: Opaque
+data:
+  shared_key: <your_shared_key> # <1>
+----
+<1> Must contain a primary or secondary key for the link:https://learn.microsoft.com/en-us/azure/azure-monitor/logs/log-analytics-workspace-overview[Log Analytics workspace] making the request.
+
+.. To obtain a link:https://learn.microsoft.com/en-us/rest/api/storageservices/authorize-with-shared-key[shared key], you can use this command in Azure CLI:
++
+[source,text]
+----
+Get-AzOperationalInsightsWorkspaceSharedKey -ResourceGroupName "<resource_name>" -Name "<workspace_name>”
```

Comment:
```suggestion
Get-AzOperationalInsightsWorkspaceSharedKey -ResourceGroupName "<resource_name>" -Name "<workspace_name>”
```

Does not render correctly in the preview because the text block is far too long and overlaps the blue menu to the right.